hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -437,14 +442,14 @@ private Constants() {
 
   /**
    * Used to specify which AWS KMS key to use if
-   * {@link #SERVER_SIDE_ENCRYPTION_ALGORITHM} is
+   * {@link #S3_ENCRYPTION_ALGORITHM} is
    * {@code SSE-KMS} (will default to aws/s3
    * master key if left blank).
    * With with {@code SSE_C}, the base-64 encoded AES 256 key.
    * May be set within a JCEKS file.
    * Value: "{@value}".
    */
-  public static final String SERVER_SIDE_ENCRYPTION_KEY =
+  public static final String S3_ENCRYPTION_KEY =
```

Comment:
Constants.java is tagged public/evolving; we can'd remove constants.

can you reinstate the original constants. you could just make it an @Deprecated reference to the new one, but it MUST still be there